docs/4.1. Behaviour - RTP Transport Type.md

```diff
@@ -61,6 +61,16 @@ Senders and Receiver Connection Management endpoints must support a core paramet
 *   `rtcp_destination_port`
 *   `rtcp_source_port`
 
+## Use of auto
```

Comment:
What do you think of adding a "Use of auto" section to each 4.x document even if it just needs the first sentence "Parameters which support use of `auto` are identified in the schemas."?

Reply:
Yeah can do